pgbouncer-cloudnative-pg.yaml

```diff
@@ -0,0 +1,53 @@
+package:
+  name: pgbouncer-cloudnative-pg
+  version: 1.22.1.15
+  epoch: 0
+  description: CloudNativePG containing PgBouncer
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - pgbouncer
+
+var-transforms:
+  - from: ${{package.version}}
+    match: '\.(\d+)$'
+    replace: '-$1'
+    to: mangled-package-version
+
+pipeline:
+  # Use fetch instead of git-checkout, if we get the repo from git we need to generate docs
+  # and the docs require pandoc which requires haskell
+  - uses: git-checkout
+    with:
+      repository: https://github.com/cloudnative-pg/pgbouncer-containers
+      expected-commit: d1bcb28762f25d48330cd2de9375cdb4c488a393
+      tag: v${{vars.mangled-package-version}}
+
+  - runs: |
+      mkdir -p "${{targets.contextdir}}"/var/log/pgbouncer
+      mkdir -p "${{targets.contextdir}}"/var/run/pgbouncer
+      mkdir -p "${{targets.contextdir}}"/etc/pgbouncer
+
+      mv /etc/pgbouncer/pgbouncer.ini "${{targets.contextdir}}"/etc/pgbouncer/pgbouncer.ini.example
+      mv /etc/pgbouncer/userlist.txt "${{targets.contextdir}}"/etc/pgbouncer/userlist.txt.example
```

Comment:
Are these example files used anywhere at runtime?

Reply:
I'm not sure but the Dockerfile says we should do that.

- https://github.com/cloudnative-pg/pgbouncer-containers/blob/main/Dockerfile#L65-L66

Reply:
similar to above, also caught my eye, but does match the upstream image. I did see another image which had a launch script which copied / cloned example configs and transformed them, not sure if the same here but i think its good to align with the upstream image